nemo/collections/tts/models/magpietts.py

```diff
@@ -226,6 +226,10 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
         if alignment_encoder_loss_scale > 0.0:
             self.alignment_encoder_loss = ForwardSumLoss(loss_scale=alignment_encoder_loss_scale)
 
+    def freeze_model(self, model):
+        for param in model.parameters():
+            param.requires_grad = False
+
```

Comment:
these lines are not necessary since Lightning does requires_grad = False and self.eval(). Pls check https://github.com/blisc/NeMo/pull/50

Reply:
Thanks! I have removed this function and I am now using lightning freeze() instead.